lib/uniffi-zcash/Cargo.toml

```diff
@@ -21,6 +21,7 @@ zcash_primitives = { version = "0.10.0", features = ["transparent-inputs"] }
 zcash_client_backend = { version = "0.7.0", features = ["transparent-inputs", "unstable"] }
 orchard = {version= "0.3.0"}
 secp256k1 = {version = "0.21.3"}
+jubjub = {version = "0.9.0"}
```

Comment:
Adding this elliptic curve crate, as the Zcash Api re-exposes the types, should be treated as part of the Zcash public API.